### PR TITLE
Implement method convert_queue_status for torque

### PR DESCRIPTION
**Pull request by ligerzero-ai**
Implements queue_table functionality for torque for remote submission purposes

### CODE REVIEW

**Review by jan-janssen — CHANGES_REQUESTED** [review]
@ligerzero-ai can you add a test for the `convert_queue_status()` function? Just provide a set of example inputs and check they are parsed correctly.

File: pysqa/wrapper/torque.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
 from pysqa.wrapper.generic import SchedulerCommands
+import subprocess
```
Comment (jan-janssen) [inline pysqa/wrapper/torque.py:6]:
```suggestion
```

**Comment by ligerzero-ai** [comment]
> @ligerzero-ai can you add a test for the `convert_queue_status()` function? Just provide a set of example inputs and check they are parsed correctly.

Should be done now

**Comment by ligerzero-ai** [comment]
Slurm is missing tests, actually. There is only a test for SGE as far as I can tell. We should add it at some point - I'll open an issue as a reminder

**Review by jan-janssen — APPROVED** [review]
Looks good to me

File: pysqa/wrapper/torque.py [hunk 2]
```diff
@@ -38,3 +40,52 @@ def get_job_id_from_output(queue_submit_output):
...
+
+        # concatenate all lines into a single string
+        input_string = "".join(lines)
+        # remove all whitespaces
+        input_string = "".join(input_string.split())
+
+        # Extract the job ID, user, job name, status, and working directory for each running job
+        regex_pattern_job_id = r"JobId:(.*?)Job_Name"
+        job_id_lst = re.findall(regex_pattern_job_id, input_string)
+        job_id_lst = [job_id_str.split(sep=".")[0] for job_id_str in job_id_lst]
```
Comment (jan-janssen) [inline pysqa/wrapper/torque.py:60]:
```suggestion
       job_id_lst = [int(job_id_str.split(sep=".")[0]) for job_id_str in job_id_lst]
```

**Pull request merged**